src/makeCalculator.js

```diff
@@ -38,6 +38,45 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+    add(number1, number2) {
```

Comment:
```suggestion
    result: 0,
    
    add(number1, number2) {
```
add space between object properties and methods and just between methods